pkg/controller/node/ipam/cloud_cidr_allocator.go

```diff
@@ -36,11 +36,12 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	v1node "k8s.io/kubernetes/pkg/api/v1/node"
 	"k8s.io/kubernetes/pkg/cloudprovider"
 	"k8s.io/kubernetes/pkg/cloudprovider/providers/gce"
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/node/util"
-	nodeutil "k8s.io/kubernetes/pkg/util/node"
+	utilnode "k8s.io/kubernetes/pkg/util/node"
```

Comment:
Why are you renaming this to utilnode?

Reply:
That's the convention we're using in the whole codebase - the alias for package is suffix of a path (without '/').

Reply:
You might then want to also change it in range_allocator.go then.
Btw - there are many other places in the code where we're using 'nodeutil'. Guess they can be changed as part of some bigger cleanup then.

Reply:
Let's not change more in this PR - we are already in code freeze - let's only fix bugs here.